romana-install/tests.devstack.yml

```diff
@@ -22,3 +22,4 @@
 - hosts: devstack_controller
   tasks:
     - shell: chdir="{{ test_dir }}" {{ test_script }} -l openstack -l endpoints >> {{ test_logfile }}
+    - shell: chdir="{{ test_dir }}" {{ test_script }} -l openstack -l e2e >> {{ test_logfile }}
```

Comment:
There's three parts:
- `clustertests` script
- a config file (eg: `test/openstack-cluster/tests`)
- the command-line that uses both.

When clustertests is run, it parses the file (list of tests with labels, commands etc), then runs only the tests matching the labels from the command-line.